pi_server/pi_server/routes_app.py

```diff
@@ -0,0 +1,74 @@
+# from pi_server import domain

```

Comment:
Why would we merge that? There's only commented code here... We should keep only what's useful for our use-case.